app/views/public/_profiletab.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="card p-1 pl-3 pr-3 mb-3">
+  <div class="row">
+    <div class="col-10 p-2">
+      <ul class="nav">
+        <li class="tab-nav-link active"><a data-toggle="tab" href="#my_posts"><i class='fa fa-handshake-o'></i> My Posts</a></li>
+        <li class="tab-nav-link"><a data-toggle="tab" href="#my_comments"><i class='	fa fa-users'></i>My comments</a></li>
+        <li class="tab-nav-link"><a data-toggle="tab" href="#my_subscriber"><i class='fa fa-comment'></i> Joined communities</a></li>
+        <li class="tab-nav-link"><a data-toggle="tab" href="#my_description"><i class='	fa fa-vcard'></i> My Description</a></li>
+        <li class="tab-nav-link"><a data-toggle="tab" href="#saved_posts"><i class='	fa fa-bookmark-o'></i> Saved Posts</a></li>
+        <li class="tab-nav-link"><a data-toggle="tab" href="#my_community"><i class='	fa fa-television'></i> My Community</a></li>
```

Comment:
all these li seems to have same code in it, can we refactor this to avoid duplicates?